### PR TITLE
Add swagger-resources endpoint

### LINKED ISSUE #311: Serve swagger on /admin/api-docs
##### **Overview of the feature request**

Today swagger is only serve on /swagger 

##### **Related issues or PR**

-   [x] Checking this box is mandatory (this is just to show you read everything)

### PR DESCRIPTION
**Pull request by EnmanuelReyes**
-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

Screen Shot 2020-10-05 at 8 40 31 PM

closes #311

### CODE REVIEW

File: generators/server/templates/dotnetcore/src/Project/Controllers/SwaggerController.cs.ejs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        {
+            _log.LogDebug("REST request to get Swagger Resources");
+            SwaggerResource sr1 = new SwaggerResource() { Name = "<%=pascalizedBaseName%>", Location = "/v2/api-docs" };
+            List<SwaggerResource> swaggerResources = new List<SwaggerResource>();
+            swaggerResources.Add(sr1);
+            return Ok(swaggerResources);
+        }
+    }
+
+    public class SwaggerResource
```
Comment (nicolas63) [inline generators/server/templates/dotnetcore/src/Project/Controllers/SwaggerController.cs.ejs:43]: Can you extract this class and put it in DTO project please ?

**Review by nicolas63 — CHANGES_REQUESTED** [review]

File: generators/server/templates/dotnetcore/src/Project/Controllers/SwaggerController.cs.ejs (hunk 1 above)
Comment (EnmanuelReyes) [reply generators/server/templates/dotnetcore/src/Project/Controllers/SwaggerController.cs.ejs:43]: Sure!
I Will do it.

Comment (EnmanuelReyes) [reply generators/server/templates/dotnetcore/src/Project/Controllers/SwaggerController.cs.ejs]: @nicolas63 Done!

**Review by nicolas63 — APPROVED** [review]
Nice work ! Thanks

**Pull request merged**